packages/gafl-webapp-service/src/pages/summary/contact-summary/route.js

```diff
@@ -209,7 +209,8 @@ const getData = async request => {
     summaryTable: getLicenseeDetailsSummaryRows(permission, countryName, request),
     uri: {
       licenceSummary: LICENCE_SUMMARY.uri
-    }
+    },
+    isLicenceForYou: permission.isLicenceForYou
```

Comment:
Rather than passing this flag to the template, for the template to have the decision logic, the decision logic should be here, supplying the correct label.